v0/destinations/factorsai/data/FactorsAIIdentifyConfig.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "destKey": "traits",
+    "sourceKeys": "traits",
+    "required": false
+  },
+  {
+    "destKey": "type",
+    "sourceKeys": "type",
+    "required": true
+  },
+  {
+    "destKey": "userId",
+    "sourceKeys": "userId",
+    "required": false
+  },
+  {
+    "destKey": "anonymousId",
+    "sourceKeys": "anonymousId",
+    "required": false
```

Comment:
For the identify call we have marked the both `userId` and `anonymousId` requirement to false and also we don't have any check for this in `transform.js`. Can you check on this?

Reply:
there will always be one of userId or anonymousId will be present